### PR TITLE
Buildables v1.2

### PR DESCRIPTION
**Pull request by SinTwo**
Removes text2path, adds constructability to a few things, adds a locker_painter.dm, and several other fixes.

### CODE REVIEW

**Comment by Kelenius** [comment]
I think it's better to do something like #1151 before adding more buildables.

**Comment by SinTwo** [comment]
...I did? Unless you want to point out a more specific part that I didn't do.
This is mostly a fix update hence the v1.2 instead of v2.

**Comment by Kelenius** [comment]
Oh. Huh.

File: code/game/machinery/mass_driver.dm [hunk 1]
```diff
@@ -9,12 +9,40 @@
...
 	active_power_usage = 50
+	circuit = /obj/item/weapon/circuitboard/mass_driver
 
 	var/power = 1.0
 	var/code = 1.0
 	var/id = 1.0
 	var/drive_range = 50 //this is mostly irrelevant since current mass drivers throw into space, but you could make a lower-range mass driver for interstation transport or something I guess.
 
+/obj/machinery/mass_driver/map/New()
+	circuit = new circuit(src)
```
Comment (PsiOmegaDelta) [inline code/game/machinery/mass_driver.dm]: Missing call to ..()

**Pull request closed without merging**